ckks/polynomial_evaluation.go

```diff
@@ -110,44 +110,14 @@ func (eval *evaluator) EvaluatePoly(ct0 *Ciphertext, pol *Poly, evakey *Evaluati
 // Returns an error if something is wrong with the scale.
 func (eval *evaluator) EvaluateCheby(op *Ciphertext, cheby *ChebyshevInterpolation, evakey *EvaluationKey) (opOut *Ciphertext, err error) {
 
-	if err := checkEnoughLevels(op.Level(), &cheby.Poly, 2/(cheby.b-cheby.a)); err != nil {
+	if err := checkEnoughLevels(op.Level(), &cheby.Poly, 1); err != nil {
```

Comment:
Please explain this line in a comment. Why did it went from `2/(cheby.b-cheby.a)` to `1` ?

Par of the challenge of understanding what is going on here is that `checkEnoughLevels` itself is undocumented. Can you take care of that ? Side note: shouldn't `checkEnoughLevels`take an `Operand` as input ?

Reply:
Previously it was checking if the change of variable would consume a level (i.e. if 2/(b-a) wasn't an integer). Since the user now has to do the change of variable himself, this check doesn't have to be done. Rather than entirely removing the check, I pass the value 1 (an integer), like it is done for the evaluation with a standard basis.

Reply:
I cannot locate this comment on the last commit. What about the second part of my comment about `checkEnoughLevels` and its arguments ?

Reply:
I documented the checkEnoughLevels method